manager/dispatcher/dispatcher.go

```diff
@@ -572,20 +572,37 @@ func (d *Dispatcher) Tasks(r *api.TasksRequest, stream api.Dispatcher_TasksServe
 			return err
 		}
 
-		select {
-		case event := <-nodeTasks:
-			switch v := event.(type) {
-			case state.EventCreateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventUpdateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventDeleteTask:
-				delete(tasksMap, v.Task.ID)
+		// bursty events should be processed in batches and sent out snapshot
+		const eventBatchLimit = 200
```

Comment:
How is this selected? This isn't really how task assignment sets work. If you want for another time block to send, you'll still send the whole batch. All this does is set the task start parallelism.

We should probably set this value to optimal limit the total number of starting tasks in a time slice.